apps/container/src/components/ConceptsHorizontalChart.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="grading">
+    <el-tooltip
+      v-for="grade in orderedDistribution"
+      placement="top"
+      :key="grade.conceito"
+      :hide-after="0"
+      :content="`
+        ${grade.conceito}: ${grades[grade.conceito].toFixed(1)}% (${
+          grade.count
+        } notas)`"
+    >
+      <span
+        class="grading-segment"
+        :class="grade.count < unthrustableThreshold ? 'unthrustable' : ''"
```

Comment:
acho que quis dizer `untrustable`, está cheio de `unthrustable` no arquivo, de uma olhada e corrija se for o caso

Reply:
Não tem propulsão pô